pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -368,12 +368,50 @@ func startDnsmasq(bridgeName string) {
 	}
 }
 
-//    pkill -u nobody -f dnsmasq.${BRIDGENAME}.conf
+//    kill -9 (-0) pid of dnsmasq on bridge
```

Comment:
No need to kill -9. Normal works just fine. And we want the logs from it exiting which we don't get with a -9